pyquil/device/_specs.py

```diff
@@ -249,7 +295,12 @@ def from_dict(d):
                                           fBellState=especs.get('fBellState'),
                                           fCZ=especs.get('fCZ'),
                                           fCZ_std_err=especs.get('fCZ_std_err'),
-                                          fCPHASE=especs.get('fCPHASE'))
+                                          fCPHASE=especs.get('fCPHASE'),
+                                          fCPHASE_std_err=especs.get('fCPHASE_std_err'),
+                                          fXY=especs.get('fXY'),
+                                          fXY_std_err=especs.get('fXY_std_err'),
+                                          fISWAP=especs.get('fISWAP'),
+                                          fISWAP_std_err=especs.get('fISWAP_std_err'))
                                 for e, especs in d["2Q"].items()],
                                key=lambda edge_specs: edge_specs.targets)
         )
```

Comment:
@ecpeterson missed one more thing 😬 
Line 319 should be
```
 especs = [EdgeSpecs(targets=(q1, q2), fBellState=0.90, fCZ=0.90, fCZ_std_err=0.05, fCPHASE=0.80, fXY=0.80, fISWAP=0.80)
              for q1, q2 in graph.edges]
```

Reply:
https://github.com/rigetti/pyquil/pull/1111